tensorrt_llm/_torch/models/modeling_qwen3_next.py

```
@@ -1251,6 +1251,11 @@ def __init__(
         )
         self.preload_weight_modules = self.model.preload_weight_modules
 
+    @classmethod
+    def get_model_defaults(cls, llm_args: 'BaseLlmArgs') -> dict:
```

Ruff (0.14.14)

[warning] Unused class method argument: `llm_args`

(ARG003)